scripts/run-external.sh

```diff
@@ -6,15 +6,49 @@ set -o pipefail
 # error on unset variables
 set -u
 
-if [[ -z "${1:-}" ]]; then
-	echo "ERROR: missing cluster name"
-	echo "usage: ❯ $0 <clustername>"
-	exit 1
-fi
-
-apiserver=$(kubectl config view -o jsonpath="{.clusters[?(@.name == \"$1\")].cluster.server}")
-cafile=$(kubectl config view -o jsonpath="{.clusters[?(@.name == \"$1\")].cluster.certificate-authority}")
-certfile=$(kubectl config view -o jsonpath="{.users[?(@.name == \"$1\")].user.client-certificate}")
-keyfile=$(kubectl config view -o jsonpath="{.users[?(@.name == \"$1\")].user.client-key}")
-
-./operator --apiserver="${apiserver}" --ca-file="${cafile}" --cert-file="${certfile}" --key-file="${keyfile}"
+run() {
+	echo " ❯ $* "
+	"$@"
+}
+
+update_crds() {
+	kubectl replace -f example/prometheus-operator-crd-full ||
+		kubectl create -f example/prometheus-operator-crd-full
```

Comment:
I'd usually do
````
kubectl apply --server-side --force-conflicts -f example/prometheus-operator-crd-full
````

Reply:
Thanks, will change to that. I thought `apply` might sometime fail if the annotations it adds is huge. Didn't know about `--server-side`